client/dashboard/sites/features.ts

```diff
@@ -82,3 +82,7 @@ export function canCreateStagingSite( site: Site ) {
 		! site?.options?.wpcom_staging_blog_ids?.length
 	);
 }
+
+export function canViewSecuritySettings( site: Site ) {
```

Comment:
Let's just inline the function. See this refactor for context:

- https://github.com/Automattic/wp-calypso/pull/104822